pkg/install/install.go

```diff
@@ -34,6 +34,7 @@ func CmdUninstall() *cobra.Command {
 	}
 	cmd.Flags().Bool("cleanup", false, "Enable deletion of Namespace and CRD's")
 	cmd.Flags().Bool("cleanup_data", false, "Clean object buckets")
+	cmd.Flags().Bool("cleanscc", false, "Enable deletion of the NooBaa SCC")
```

Comment:
why not include in cleanup? it cleans other clusterwide entities

Reply:
IDK just for a shared cluster like gogo - I accidentally deleted the SCC there